common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -20,8 +30,16 @@ export default leafCommand(commandInfo, async (options) => {
   const mochaArgs = updatedArgs?.length
     ? updatedArgs?.join(" ")
     : '--timeout 1200000 --exclude "test/**/browser/*.spec.ts" "test/**/*.spec.ts"';
-  return runTestsWithProxyTool({
+  const command = {
     command: `mocha ${defaultMochaArgs} ${mochaArgs}`,
-    name: "node-tests",
-  });
+    name: "node-tests-ts",
```

Comment:
```suggestion
    name: "node-tests",
```
Or add `-js` to the other one for consistency.